contrib/inventory/vagrant.py

```diff
@@ -74,7 +74,7 @@ def get_ssh_config():
 
 # list all the running boxes
 def list_running_boxes():
-    output = subprocess.check_output(["vagrant", "status"]).split('\n')
+    output = subprocess.check_output(["vagrant", "status"]).decode('utf-8').split('\n')
```

Comment:
What do you think of using`splitlines()` over `split()`? I think it would be a better option here, since it handles the concept of universal newlines, thus this would be fine on Windows too, if such support is required.